### PR TITLE
Avoid some unnecessary calls to OrderedDict.update and simplify

### PR DESCRIPTION
**Pull request by juanchopanza**
min/max handling of normal distribution.

### CODE REVIEW

File: examples/synthesis_json.py [hunk 1]
```diff
@@ -91,27 +91,28 @@ def transform_distribution(data, datamin=None, datamax=None):
...
 
-    if data["type"] == 'norm':
-        data_dict.update({"type": "normal"})
-        data_dict.update({"mu": data["params"][0]})
-        data_dict.update({"sigma": data["params"][1]})
+    if data["type"] == 'uniform':
+        data_dict.update({"type": "uniform",
+                          "min": np.min(data["params"]),
+                          "max": np.max(data["params"])})
+        return data_dict
```
Comment (juanchopanza) [inline examples/synthesis_json.py]: The logic here is that min/max are the parameters of the uniform distribution, so we ignore datamin/datamax.

File: examples/synthesis_json.py [hunk 2]
```diff
@@ -91,27 +91,28 @@ def transform_distribution(data, datamin=None, datamax=None):
...
     data_dict = OrderedDict()
 
-    if data["type"] == 'norm':
-        data_dict.update({"type": "normal"})
-        data_dict.update({"mu": data["params"][0]})
-        data_dict.update({"sigma": data["params"][1]})
+    if data["type"] == 'uniform':
+        data_dict.update({"type": "uniform",
+                          "min": np.min(data["params"]),
+                          "max": np.max(data["params"])})
```
Comment (juanchopanza) [inline examples/synthesis_json.py]: This use of `np,min`, `np.max` is taken from #114

**Comment by juanchopanza** [comment]
That is fine, because JSON objects are unordered anyway. But it might be nice to keep an order for presentation purposes. I'll change this.

**Comment by juanchopanza** [comment]
Is this OK?

**Comment by lidakanari** [comment]
This is ok I guess, if the output that it produces is good. However, you will have to show me how to erase my last commits without breaking the merge!

**Pull request merged**